lua/autorun/server/sv_caf_autostart.lua

```diff
@@ -121,9 +121,9 @@ local function LoadAddonStatus( addon, defaultstatus )
 end
 
 local function OnEntitySpawn(ent , enttype , ply)
-	if ent == NULL then
+	if ent:IsValid() ~= true then
```

Comment:
I haven't programmed for garrysmod in a while, but wouldn't this raise a error if ent is really NULL for whatever reason?

OnEntitySpawn is called by garrysmod hooks for PlayerSpawnedNPC, PlayerInitialSpawn, PlayerSpawnedProp, PlayerSpawnedSENT, PlayerSpawnedVehicle

Reply:
Why not just replace ent:IsValid() by IsValid(ent) which according to the wiki nilchecks + validchecks

Reply:
> I haven't programmed for garrysmod in a while, but wouldn't this raise a error if ent is really NULL for whatever reason?

LUA doesn't null, it nil's.
Either way, I will switch it to IsValid(ent)